packages/sanity/src/core/form/types/inputProps.ts

```diff
@@ -77,12 +79,21 @@ export interface ObjectInputProps<
   /** @beta */
   onFieldClose: (fieldName: string) => void
 
+  /** @beta */
+  renderAnnotation: RenderAnnotationCallback
+
+  /** @beta */
+  renderBlock: RenderBlockCallback
```

Comment:
The introduction of these callbacks is a breaking change, so we need to find a way to make them optional

Reply:
I had a small question which might have been answered in separate conversations before: Do we have a reason why these new callbacks need to be mandatory?

Reply:
I made them mandatory to follow the same patterns as with `renderItem` which is mandatory. All of these should probably be optional at some point, but we can change that later.